packages/web/src/components/EventsList/EventsList.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Event, eventStyle } from './utils';
+import { Badge } from '@chakra-ui/react';
+
+const Event = ({ event }: { event: Event }) => {
+  
+  const { name , description, start, end } = event;
+  const stringFormat: Intl.DateTimeFormatOptions = { hour: 'numeric', minute: 'numeric' };
+  const formattedStart = new Date(start).toLocaleTimeString([], stringFormat);
+  const formattedEnd = new Date(end).toLocaleTimeString([], stringFormat);
+  
+  const now = new Date();
+  const _start = new Date(start);
+  const _end = new Date(end);
+  const inProgress = _start < now && now < _end;
+  const past = now > _end;
+  const badge = inProgress ? 'IN PROGRESS' : past ? 'PAST' : '';
+  
+  return (
+    <li style={eventStyle(badge)}>
+      <h2>
+        {name}&nbsp;&nbsp;
+        
+        { (badge=='IN PROGRESS') &&
+          <Badge colorScheme="green" variant="solid" style={{ float:'right' }}>
+            {badge} &nbsp;&nbsp;
+            {/* clock icon */}
+            &#x23F0;
```

Comment:
This is perfect for `<Flex direction="row" gap={2}>` and just use the straight emoji instead of the unicode; it renders the same and is more readable in code